src/pynwb/testing/mock/ecephys.py

```diff
@@ -119,3 +120,34 @@ def mock_SpikeEventSeries(
         nwbfile.add_acquisition(spike_event_series)
 
     return spike_event_series
+
+
+def mock_Units(
+    num_units: int = 10,
+    max_spikes_per_unit: int = 10,
+    seed: Optional[int] = None,
+    nwbfile: Optional[NWBFile] = None,
+) -> Units:
+
+    units_table = Units()
+    units_table.add_column(name="unit_name", description="a readable identifier for the unit")
+
+    if seed is None:
+        seed = 0
```

Comment:
```suggestion
    seed: int = 0,
    nwbfile: Optional[NWBFile] = None,
) -> Units:

    units_table = Units()
    units_table.add_column(name="unit_name", description="a readable identifier for the unit")

```

Reply:
Not that it matters much in this case but I am following numpy convention for default seeds.

https://numpy.org/doc/stable/reference/random/generator.html#numpy.random.default_rng

I think the semantics are better and leaves the possibility of changing the initialization if the seed is not appropriate.

Reply:
`None` makes sense in their case because they are not using the 0 seed:

> If None, then fresh, unpredictable entropy will be pulled from the OS.

Reply:
Good point. 

Do you think passing the seed as None directly would be better or do you prefer fix by default as you are proposing in your change?

Reply:
This library will be used mostly for testing, so it may make sense to default to 0 for reproducibility. I think defaulting to None and passing that directly would be fine too.

Reply:
Let's go with your proposed change then. Thanks for the feedback.